### PR TITLE
Mention macports installation procedure on macOS

### PR DESCRIPTION
**Pull request by dgsb**
The version of `golangci-lint` has been updated to the v1.30 version in macports. https://github.com/macports/macports-ports/commit/3703aa3bcc8fd46f8425947ca578e627936e2a30

### CODE REVIEW

File: docs/src/docs/usage/install/index.mdx [hunk 1]
```diff
@@ -53,6 +53,12 @@ brew install golangci/tap/golangci-lint
 brew upgrade golangci/tap/golangci-lint
 ```
 
+It can also be installed through [macport](https://www.macports.org/)
```
Comment (sayboras) [inline docs/src/docs/usage/install/index.mdx:56]: I am not so sure if it's good point to mention macport here, as I don't think any one from @golangci/team will send PR to keep version in sync with latest release. 

Maybe just add some note saying that macport is community driven, and not official maintained by golangci team.

**Review by sayboras — CHANGES_REQUESTED** [review]

File: docs/src/docs/usage/install/index.mdx (hunk 1 above)
Comment (dgsb) [reply docs/src/docs/usage/install/index.mdx:56]: Ok, I will indicate that this method is not officially supported by golangci-lint team

Comment (dgsb) [reply docs/src/docs/usage/install/index.mdx]: @sayboras I've done the requested documentation change

Comment (sayboras) [reply docs/src/docs/usage/install/index.mdx]: Thanks. LGTM :)

**Review by sayboras — APPROVED** [review]
LGTM 💯

**Comment by dgsb** [comment]
up ?

**Comment by sayboras** [comment]
Seems like there is no more comment. Merging this :100:

**Pull request merged**